assesment.js

```diff
@@ -0,0 +1,82 @@
+document.addEventListener("DOMContentLoaded",() =>{
+    console.log("page loaded");
+    fetchMovies()
+    SelectMovie()
+
+})
+
+function fetchMovies(){
+    const url = "https://ghibliapi.herokuapp.com/films"
+    // axios.get(url).then((response)=>{
+    //     response.data
+    //     })
+    
+    fetch(url).then(response =>{
+        return response.json() 
+        //console.log(response)
+    })
+    .then(data => {
+        let pick = data
+        console.log(pick[0])
+        let selection = document.querySelector("select");
+        
+        for(movie in pick){
+            let title = document.createElement("option");
+            title.value = pick[movie].title;
+            title.innerText = pick[movie].title;
+            selection.appendChild(title);
+
+        }
+    })
+}
+
+function SelectMovie(){
```

Comment:
It is not good practice to have variable names starting with uppercase. Leave those only for Classes or constructor functions.